### PR TITLE
Assessment add metrics

### PR DESCRIPTION
**Pull request by dv3-000**
Very small PR, just adds High Score and Generation High Score to the generated "ga_data.csv" file

### CODE REVIEW

**Review by davidd-55 — APPROVED** [review]
LGTM!

File: helpers/snakeGameGATrain.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def game_over(self):
 			#Write data about this generation to ga_data.csv
 			csv_file = open("ga_data.csv", "a+")
 			csv_writer = csv.writer(csv_file)
-			csv_writer.writerow([date_time, str(self.num_generations),str(best_fitness),str(average_fitness),str(average_game_score)])
+			csv_writer.writerow([date_time, str(self.num_generations),str(best_fitness),str(average_fitness),str(average_game_score), str(self.high_score), str(high_score_per_cur_gen)])
```
Comment (jackdavidweber) [inline helpers/snakeGameGATrain.py:118]: Do we want to add high score and high score per cur gen into GAdata.txt just to make it consistent?

Comment (jackdavidweber) [reply helpers/snakeGameGATrain.py:118]: Alternatively might consider simply getting rid of everything except for datetime and binary representation of the generation from GAdata.txt if we are never going to use it and we can link back with the datetime

Comment (jackdavidweber) [reply helpers/snakeGameGATrain.py:118]: Any opinions?

Comment (dv3-000) [reply helpers/snakeGameGATrain.py:118]: Personally idk why we even need GAdata.txt, we could just add the binary to the .csv and deprecate GAdata.txt

Comment (dv3-000) [reply helpers/snakeGameGATrain.py:118]: would serve the same exact purpose but just be cleaner, easier to manipulate, and all in one place

**Pull request merged**